Oyinsplash/PropertyPro-Lite
# feature(signin): Add signin

#### What does this PR do?
- Add signin form to the homepage.
- Add how it works section to the homepage.
- Add footer to the homepage.

#### Description of Task to be completed?
Carry out the following Tasks:
- Create a How It Works section on index.html
- Create a Signin section after the How It Works section
- Add a Signin form in the Signin section.
- Create a Signin button to submit the sign in credentials of a user.
- Connect the Signin link at the top right corner of the homepage to the Signin section.

#### How should this be manually tested?
- Clone repo and change into the working directory.
- Switch to the ft-UI-signin-166658790 branch and launch index.html.
- Click on the Sign in link at the top right corner of homepage.
- Observe the signin form and signin button.

#### What are the relevant pivotal tracker stories?
#166658790

#### Screenshot
![image](https://user-images.githubusercontent.com/46001371/59976041-15defd80-95b7-11e9-9de9-f7f920f7fd1d.png)
![image](https://user-images.githubusercontent.com/46001371/59976051-3dce6100-95b7-11e9-9564-f2fd5775fe5a.png)
![image](https://user-images.githubusercontent.com/46001371/59976056-50e13100-95b7-11e9-8d38-a4b755fb6257.png)
